Staging for platforms whitepaper v1.1 update (#77)
Cloud-Native-Platform-Engineering/cnpe-community: platforms-whitepaper/latest/CHANGELOG.md

# Platforms Whitepaper Changelog

## [v1.1]

### Summary

> This release represents a significant revision of the CNCF Platforms White Paper, refining the narrative framing, restructuring key sections, and updating content to reflect the current state of platform engineering practice.

Reviewer: I feel like using this phrasing should be revisited. If we think it is significant, then we should say it and think about the version numbering. But IMHO something like this may be more accurate:

```suggestion
This release represents the first investment initiative in updating the CNCF Platforms White Paper. As a minor release, the goal is to refine the content to reflect the current state of platform engineering practice while leaving larger improvements to the upcoming v2.0.

This is a minor release clarify the narrative framing, restructure key sections for readability, and update content to reflect the Platform Engineering Maturity Model work which was released after this white paper.
```

Reviewer 2: Chatted in the call after you jumped off: It could be helpful to use "initiative" here instead of wave, with the same spirit, but also to show that we're consciously trying to follow the TOC's guidance on having planned/scoped initiatives.

Reviewer: updated!

---

### Introduction

> - Reframed the opening to position platform engineering as an optimisation for enterprises that experienced DevOps autonomy but need to manage the resulting costs, security risks, and inefficiencies.

Reviewer 3: I noticed we already use "optimizes" elsewhere in the document. How about we standardize on "optimization" here for consistency?

```suggestion
- Reframed the opening to position platform engineering as an optimization for enterprises that experienced DevOps autonomy but need to manage the resulting costs, security risks, and inefficiencies.
```

### Attributes of Platforms

- **User experience**: Added **AI agents** as a supported interface type alongside GUIs, APIs, CLIs, IDEs, and portals.
- **Self-service**: Clarified that capabilities should be available on-demand without waiting for a manual review or approval.
- **Secure by default**: Renamed to **"Secure and compliant by default"**; expanded description to note that security, governance, and compliance requirements should be baked into the platform to reduce user cognitive burden while ensuring consistent enforcement.

### Attributes of Platform Teams

- Updated the list of platform team responsibilities to include **AI agents** alongside portals, APIs, documentation, templates, and CLI tools.
- Minor formatting cleanup (em dash usage, golden paths terminology).

### Challenges with Platforms

- Restructured from a brief bullet list into three explicitly named challenge sections with expanded guidance:
  1. **The platform is disconnected from the actual needs and wants of developers** - Emphasises product management partnership and the danger of top-down mandates.

>   2. **Too much effort is spent on low-value solutions that are overfitted to early-adopters** - Advises maintaining a birds-eye roadmap view while cherishing early adopters as champions.

Reviewer 3: Minor grammar fix: "bird's-eye" requires an apostrophe.

```suggestion
  2. **Too much effort is spent on low-value solutions that are overfitted to early-adopters** - Advises maintaining a bird's-eye roadmap view while cherishing early adopters as champions.
```
